index.html

```diff
@@ -46,15 +46,16 @@
     <link rel="stylesheet" href="https://rsms.me/inter/inter.css" />
     <link href="/dist/output.css" rel="stylesheet" />
   </head>
-  <!-- overflow-y-hidden doesnt work with safari -->
+  <!-- overflow-y-hidden doesnt work with safari, only overflow-hidden -->
   <body
     x-data="{
       characterMenuOpen: false,
     }"
+    class="overflow-hidden flex flex-col"
   >
     <nav
       id="nav"
-      class="flex items-center px-4 py-2 gap-2 sticky top-0 bg-white z-10"
+      class="flex items-center px-4 py-2 gap-2 sticky bg-white z-10"
```

Comment:
removed `top-0` here since this elem wasnt positioned so it shouldnt do anything?